build/project-template/__PROJECT_NAME__/build.xcconfig

```diff
@@ -2,4 +2,8 @@
 //     security find-identity |  sed -n 's/.*\("[^"]*"\).*/\1/p' | grep 'iPhone'
 
 // generic 'iPhone Developer' (no quotes) will match the right Identity with the right Provisioning Profile plus Certificate, based on the app bundle id
-CODE_SIGN_IDENTITY = iPhone Developer
+
+// TNS_TYPESCRIPT_DECLARATIONS_PATH = $SOURCE_ROOT/TypeScript Declarations
+// TNS_DEBUG_METADATA_PATH = $SOURCE_ROOT/Debug Metadata
+
+CODE_SIGN_IDENTITY = iPhone Developer
```

Comment:
It would be better to keep `CODE_SIGN_IDENTITY` close to the comment explaining how to obtain it above.